pdc/apps/compose/models.py

```diff
@@ -505,3 +505,41 @@ class Meta:
         unique_together = (
             ("variant_arch", "image"),
         )
+
+
+class Location(models.Model):
+    name = models.CharField(max_length=50)
+    short = models.CharField(max_length=50, unique=True)
+
+
+class Scheme(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+
+
+class ComposeTree(models.Model):
+    compose             = models.ForeignKey("Compose")
+    variant             = models.ForeignKey("Variant")
+    arch                = models.CharField(max_length=50)
+    location            = models.ForeignKey("Location")
+    scheme              = models.ForeignKey("Scheme")
+    url                 = models.CharField(max_length=100)
```

Comment:
max length 100 is definitely too restrictive. Why should the url be that short? We would certainly hit this limit